src/cmd/exporter/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+	"strings"
+
+	_ "github.com/lib/pq"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/spf13/viper"
+
+	"github.com/goharbor/harbor/src/common/dao"
+	commonthttp "github.com/goharbor/harbor/src/common/http"
+	"github.com/goharbor/harbor/src/common/models"
+	"github.com/goharbor/harbor/src/lib/log"
+	"github.com/goharbor/harbor/src/pkg/exporter"
+)
+
+func main() {
+	viper.AutomaticEnv()
+	viper.SetEnvPrefix("harbor")
+	viper.AutomaticEnv()
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	dbCfg := &models.Database{
+		Type: "postgresql",
+		PostGreSQL: &models.PostGreSQL{
+			Host:         viper.GetString("database.host"),
+			Port:         viper.GetInt("database.port"),
+			Username:     viper.GetString("database.username"),
+			Password:     viper.GetString("database.password"),
+			Database:     viper.GetString("database.dbname"),
+			SSLMode:      viper.GetString("database.sslmode"),
+			MaxIdleConns: viper.GetInt("database.max_idle_conns"),
+			MaxOpenConns: viper.GetInt("database.max_open_conns"),
+		},
+	}
+	if err := dao.InitDatabase(dbCfg); err != nil {
```

Comment:
Set the number of conns to relatively low, we don't expect the exportor to use many connections